contrib/fwupd.spec.in

```diff
@@ -453,6 +451,8 @@ done
 %{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_linux_swap.so
 %{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_linux_tainted.so
 %if 0%{?have_msr}
+%{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_acpi_dmar.so
+%{_libdir}/fwupd-plugins-%{fwupdplugin_version}/libfu_plugin_acpi_ivrs.so
```

Comment:
Nah, `have_msr` is defined for i686 and x86_64

Reply:
Oh, fedora actually builds 32 bit too?  I guess technically these tables can be there.  I'll fixup the meson build.

Reply:
Fixed